### ISSUE TITLE
Problems with Scene integration

### ISSUE THREAD

**Issue by Scaby79** [CDeenen/Soundscape#32]
As soundscape is integrated in the scene configuration you can add a soundscape in the ambient tab of the scene configuration. 
When you do, it loads perfectly, when a scene is activated but it DOESN'T STOP, when another scene is activated. 

Using a title of a playlist or a whole playlist (instead of a soundscape) on the other hand DOES STOP, when you change an active scene so I thought this is worth an issue here.

It would be very nice if that could be fixed if possible.

**Comment by CDeenen** [comment]
I just released an update and this should now be fixed

**Comment by Scaby79** [comment]
Sorry, but the problem still persists.

**Comment by Scaby79** [comment]
> I just released an update and this should now be fixed

The last release didn't fix the issue.

**Comment by CDeenen** [comment]
You're right, this isn't fixed. Very strange, I'm sure I tested it multiple times.
I'll have another look at it...

**Comment by CDeenen** [comment]
Please give the latest version another try, it hopefully works now

**Comment by Scaby79** [comment]
I just tested the Version 1.1.1 of your module in a world in FVTT version 10.291.
It is not working entirely as it should:

If you are switching to a scene where there is another soundscape set, the soundscape of the scene you are leaving is stopped as intended. 

If you are switching to a scene with no soundscape set (either because there is a single sound or a playlist  or no sound at all set via the core settings option) the soundscape is not always* stopping. 

*comment for the "not always":
The default option in the soundscape section of the scene configuration apparently is "Unchaged". With this option selected, a soundscape does NOT stop, when leaving a scene.
If the option "None" in the soundscape section of the scene configuration is set, the soundscape of the scene that is just left DOES actually stop (just as intended).
BUT: The selected option in the soundscape section of the scene configuration automatically switches back to "Unchanged" (at least when opening the configuration dialog) and even if this change isn't confrmed or saved, stopping the soundscape is not working anymore (in other words: it breaks again)!

Suggested solution:
Maybe setting the default option in the soundscape section of the scene configuration to "None" instead of "Unchanged" could solve the problem for good?

Hope that helps and enables you to fix it.

Thx for your effort. I love your module!
